### DISCUSSION TITLE
Layout

### DISCUSSION THREAD

**Discussion by MeAriJit** [clientIO/joint#2145] [Q&A] [answered]
### Introduction

Is serpentine layout comes by default lay-outing option? If yes, is it accessible through joint.layout.SerpentineLayout ?

If no, are you suggesting to follow the algorithm you written for some of the demo(serpentine layout,chatgpt demo)?

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS, JointJS+

**Comment by kumilingus** [comment, marked as answer]
Eventually, it will become part of the JointJS. Until then, please just copy and use the code from the demo.
